uplc-discharge.md

```diff
@@ -16,7 +16,11 @@ module UPLC-DISCHARGE
   
   rule discharge(< con T:TypeConstant C:Constant >) => (con T C)  
 
-  rule discharge(< lam I:UplcId T:Term RHO:Map >) => (lam I dischargeTerm(T, RHO)) 
+  rule discharge(< lam I:UplcId T:Term RHO:Map >) => (lam I dischargeTerm(T, RHO[I <- undef]))
+  requires I in_keys(RHO)
```

Comment:
I would hope that `RHO[I <- undef] ==K RHO` when `notBool(I in_keys(RHO))`, so I think the `requires` here and the other case below might not be needed?

Reply:
Yes, map delete is total. I will change it.